papis/tui/utils.py

```diff
@@ -1,6 +1,50 @@
 import re
 from typing import Optional, List, Callable, Any
 
+PAPIS_PYGMENTS_DEFAULT_STYLE = {
+    "pygments.whitespace": "ansigray",
+    "pygments.comment": "italic ansigreen",
+    "pygments.comment.preproc": "noitalic ansiyellow",
+    "pygments.keyword": "bold ansigreen",
+    "pygments.keyword.pseudo": "nobold",
+    "pygments.keyword.type": "nobold ansired",
+    "pygments.operator": "ansigray",
+    "pygments.operator.word": "bold ansimagenta",
+    "pygments.name.builtin": "ansigreen",
+    "pygments.name.function": "ansicyan",
+    "pygments.name.class": "bold ansicyan",
+    "pygments.name.namespace": "bold ansicyan",
+    "pygments.name.exception": "bold ansired",
+    "pygments.name.variable": "ansiblue",
+    "pygments.name.constant": "ansired",
+    "pygments.name.label": "ansigreen",
+    "pygments.name.entity": "bold ansigray",
+    "pygments.name.attribute": "ansigreen",
+    "pygments.name.tag": "bold ansigreen",
+    "pygments.name.decorator": "ansimagenta",
+    # Note: In Pygments, Token.String is an alias for Token.Literal.String,
```

Comment:
```suggestion
    # NOTE: In Pygments, Token.String is an alias for Token.Literal.String,
```
? I think most editors recognize the uppercase NOTE as being special (I think `pylint` even yells about `FIXME` and `TODO`).